app/routes/docs/newproducers.mdx

```diff
@@ -0,0 +1,97 @@
+---
+handle:
+  breadcrumb: New Producers
+---
+
+import {
+  ProcessList,
+  ProcessListItem,
+  ProcessListHeading,
+  SummaryBox,
+} from '@trussworks/react-uswds'
+import { GitHubLabel } from '~/components/GitHubLabel'
+
+# Setting up New GCN Notice Producers
+
+The following steps guide new instrument, mission, or observatory producers
+into setting up new notices streams that are distributed to the user community
+via [Kafka](faq#what-is-kafka). This process requires interactions
+with the [GCN Team](https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn)
+to enable these accounts and topics are setup on the GCN Kafka broker,
+and we are happy to work together to help you define schema utilizing our
+core JSON schema.
+
+## Steps to Start Publishing Alerts
+
+<ProcessList>
+   <ProcessListItem>
+     <ProcessListHeading type="h3">
+       Team Member Accounts
```

Comment:
All of the process list headings should be complete sentences, minus the terminal punctuation.